### PR TITLE
correct misinformation on default macaroon setup

### PR DESCRIPTION
**Pull request by gabbyprecious**
## Change Description
Change docs on the default configuration of macaroon.
https://github.com/lightningnetwork/lnd/issues/5444

## Steps to Test
Steps for reviewers to follow to test the change.

### Testing
- [x] Your PR passes all CI checks.
- [x] Tests covering the positive and negative (error paths) are included.
- [x] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [x] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [x] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [x] Any new logging statements use an appropriate subsystem and logging level.
- [x]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -511,7 +511,7 @@ bearer credentials allowing for delegation, attenuation, and other cool
 features. You can learn more about them in Alex Akselrod's [writeup on
 Github](https://github.com/lightningnetwork/lnd/issues/20).
 
-Running `lnd` for the first time will by default generate the `admin.macaroon`,
+Running `lncli create` to create a wallet for the first time will by default generate the `admin.macaroon`,
```
Comment (guggero) [inline docs/INSTALL.md:514]: Since you can only run `lncli create` once, maybe we could remove the "for the first time".

**Review by guggero — COMMENTED** [review]
Thanks for the fix!
Please take a look at our [contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) on how to prefix the commit message.
And you should either add an entry to the release notes or add `[skip ci]` to the commit message.

**Comment by guggero** [comment]
You can't change the commit message through the GitHub UI, in case you're trying that. Please squash and update your commits into a single one. There's a guide [here](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits).

File: docs/INSTALL.md [hunk 2]
```diff
@@ -511,8 +511,7 @@ bearer credentials allowing for delegation, attenuation, and other cool
 features. You can learn more about them in Alex Akselrod's [writeup on
 Github](https://github.com/lightningnetwork/lnd/issues/20).
 
-Running `lnd` for the first time will by default generate the `admin.macaroon`,
-`read_only.macaroon`, and `macaroons.db` files that are used to authenticate
+Running `lncli create` to create a wallet, will by default generate the `admin.macaroon`, `read_only.macaroon`, and `macaroons.db` files that are used to authenticate
```
Comment (ellemouton) [inline docs/INSTALL.md:514]: nit: please wrap at 80 chars

**Review by ellemouton — COMMENTED** [review]
Nice fix!

One nit re wrapping the text. 

>  I can't seem to get to work around the skip ci

I think it is cause you have not added a space between the last word in the commit message and the `[`. 
So instead of `lndwallet:change docs for macaroon creation[skip ci]`, try `
lndwallet: change docs for macaroon creation [skip ci]` (note the space after the `:` too)

**Review by guggero — APPROVED** [review]
LGTM, thanks!

Can you squash the two commits and prefix it with `docs: ` instead of `lnwallet` please?

**Review by ellemouton — APPROVED** [review]
LGTM!

**Pull request merged**